### PR TITLE
Improve navigation of adr plugin

### PR DESCRIPTION
**Pull request by dweber019**
Signed-off-by: David Weber <david.weber@w3tec.ch>

## Hey, I just made a Pull Request!

This PR will solve https://github.com/backstage/backstage/issues/15995

The new navigation looks like this
CleanShot 2023-01-29 at 01 01 53@2x

#### :heavy_check_mark: Checklist

- [X] A changeset describing the change and affected packages. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#creating-changesets))
- [X] Added or updated documentation
- [X] Tests for new functionality and regression tests for bug fixes
- [X] Screenshots attached (for UI changes)
- [X] All your commits have a `Signed-off-by` line in the message. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#developer-certificate-of-origin))

### CODE REVIEW

**Comment by djamaile** [comment]
pinging @kuangp to have a look

**Review by kuangp — APPROVED** [review]
Minor suggestion but overall looks good to me 🎉 

Thanks for the awesome contribution! 👏

File: plugins/adr/src/api/types.ts [hunk 1]
```diff
@@ -30,6 +30,15 @@ export type AdrFileInfo = {
...
   name: string;
+
+  /** The title of the ADR. */
+  title: string;
+
+  /** The status of the ADR. */
+  status: string;
+
+  /** The date of the ADR. */
+  date: string;
```
[the remark this answers is not in the document]
Comment (dweber019) [reply plugins/adr/src/api/types.ts:41]: Great catch. Will update the api reports too. thx

**Comment by dweber019** [comment]
@kuangp updated PR :wink:

**Review by kuangp — APPROVED** [review]
Looks good to me!

@djamaile over to you and the maintainers for final review

**Comment by dweber019** [comment]
Nice idea. Have added the screenshot in the README.

File: plugins/adr-backend/src/service/router.ts [hunk 2]
```diff
@@ -59,13 +60,20 @@ export async function createRouter(
...
+        files.map(async file => {
+          const fileContent = await file.content();
+          const adrInfo = madrParser(fileContent.toString());
+          return {
+            type: 'file',
+            name: file.path.substring(file.path.lastIndexOf('/') + 1),
+            path: file.path,
+            ...adrInfo,
+          };
+        }),
```
[the remark this answers is not in the document]
Comment (dweber019) [reply plugins/adr-backend/src/service/router.ts]: Like your way better 😉 PR updated

**Review by Rugvip — APPROVED** [review]
Looks good 👌, thank you! 👍 

Let's :shipit:

**Pull request merged**